.github/workflows/server_workflow.yml

```diff
@@ -1,6 +1,9 @@
 name: build_server
 
-on: [push, pull_request]
+on: 
```

Comment:
What is the reason behind removing the GitHub action on push? Wouldn't be better to have it on push too in order to fix whatever problem we might encounter?